packages/odyssey-storybook/src/components/odyssey-labs/GroupPicker/GroupPicker.stories.tsx

```diff
@@ -0,0 +1,204 @@
+  },
+];
+
+const storybookMeta: Meta<typeof GroupPicker> = {
+  title: "Labs Components/GroupPicker",
+  component: GroupPicker,
+  argTypes: {
+    hasMultipleChoices: {
+      control: "boolean",
+      description: "Enables multiple choice selection",
+      table: {
+        type: {
+          summary: "boolean",
+        },
+      },
+    },
+    hint: {
+      control: "text",
+      description: "The hint text for the autocomplete input",
+      table: {
+        type: {
+          summary: "string",
+        },
+      },
+    },
+    isCustomValueAllowed: {
+      control: "boolean",
+      description: "Allows the input of custom values",
+      table: {
+        type: {
+          summary: "boolean",
+        },
+      },
+    },
+    isDisabled: {
+      control: "boolean",
+      description: "Disables the autocomplete input",
+      table: {
+        type: {
+          summary: "boolean",
+        },
+      },
+    },
+    isLoading: {
+      control: "boolean",
+      description: "Displays a loading indicator",
+      table: {
+        type: {
+          summary: "boolean",
+        },
+      },
+    },
+    isReadOnly: {
+      control: "boolean",
+      description: "Makes the autocomplete input read-only",
+      table: {
+        type: {
+          summary: "boolean",
+        },
+      },
+    },
+    label: {
+      control: "text",
+      description: "The label text for the autocomplete input",
+      table: {
+        type: {
+          summary: "string",
+        },
+      },
+    },
+    onChange: {
+      control: null,
+      description:
+        "Callback fired when the value of the autocomplete input changes",
+      table: {
+        type: {
+          summary: "func",
+        },
+        defaultValue: "",
+      },
+    },
+    onInputChange: {
+      control: null,
+      description:
+        "Callback fired when the input value of the autocomplete input changes",
+      table: {
+        type: {
+          summary: "func",
+        },
+        defaultValue: "",
+      },
+    },
+    options: {
+      control: null,
+      description: "The options for the autocomplete input",
+      table: {
+        type: {
+          summary:
+            "Array<OptionType> | GroupedOptionType<OptionType>[] | Promise<Array<OptionType> | GroupedOptionType<OptionType>[]>",
+        },
+      },
+    },
+    value: {
+      control: null,
+      description: "The value of the autocomplete input",
+      table: {
+        type: {
+          summary: "OptionType | OptionType[]",
+        },
+      },
+    },
+  },
+  args: {
+    label: "Languages",
+    hint: "Languages supports in the system",
+    options: stations,
+  },
+  decorators: [MuiThemeDecorator],
+  tags: ["autodocs"],
+};
+
+export default storybookMeta;
+
+type GroupPickerType = AutocompleteProps<
+  GroupPickerOptionType,
+  boolean | undefined,
+  boolean | undefined
+>;
+
+export const GroupPickerDefault: StoryObj<GroupPickerType> = {
```

Comment:
We shouldn't need this code if we make a `GroupPickerProps` type in the `GroupPicker` component file right? Or will we still need to define it here because of the generics?

```suggestion
type GroupPickerPropsType = AutocompleteProps<
  GroupPickerOptionType,
  boolean | undefined,
  boolean | undefined
>;

export const GroupPickerDefault: StoryObj<GroupPickerPropsType> = {
```

Reply:
`GroupPickerPropsType` is needed and also replaced from `AutocompleteProps` to `GroupPickerProps`